eva-accession-release-automation/include_mapping_weight_from_dbsnp/construct_snpmapinfo_schema_bank.py

```diff
@@ -24,9 +24,12 @@
 # | apple_3750    | b149_snpmapinfo       | snp_type\nsnp_id\nweight     |                 3 |
 # +---------------+-----------------------+------------------------------+-------------------+
 
-from snpmapinfo_metadata import *
 import click
 
+from pg_query_utils import get_pg_connection_handle
+from snpmapinfo_metadata import *
+from __init__ import *
```

Comment:
I find these import confusing and are frowned upon by [PEP8](https://pep8.org/#imports). In the majority of cases here it only imports one or two functions.

Reply:
Addressed.